codice/ddf
# [2.13] DDF-4075 Adds prerequisite on catalog-core-api to catalog-security-filter

#### What does this PR do?
Adds a prerequisite to `catalog-core-api` to ensure that `catalog-security-filter` can correctly start.

#### Who is reviewing it? 
@lamhuy @brjeter 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@stustison
@tbatie

#### How should this be tested?
Full CI build will be sufficient to ensure that the system initializes correctly.


#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4075](https://codice.atlassian.net/browse/DDF-4075)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
